app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -29,17 +29,38 @@
         <p><%= title %>: <span class="other-date definition"><%= date %></span></p>
       <% end %>
     <% end %>
-    <% if history.any? %>
+    <%
+      hidden_history =
+          local_assigns.include?(:hidden_history) ? hidden_history : []
+    %>
+    <% if history.any? || hidden_history.any? %>
```

Comment:
Also, how that'd fit with the [history expansion example we have on whiteall](https://www.gov.uk/government/consultations/property-ownership-and-public-contracting-by-foreign-companies-improving-transparency)?